### PR TITLE
Fixed Wing Maps

### PR DESCRIPTION
**Pull request by acfloria**
- Add the vorab world

- Minor changes so that the fixed-wing HIL-Simulation with the planning framework is able to run.

### CODE REVIEW

**Comment by acfloria** [comment]
@philipoe can you review the changes?

**Comment by philipoe** [comment]
@hitimo Can you please review instead of me? I need to finish Sun2Ice work at the moment and will be gone at the end of the week.

File: README.md [hunk 1]
```diff
@@ -69,15 +69,29 @@ Installation Instructions
...
+   $ mkdir software
+   $ cd software
+   $ git clone https://github.com/google/protobu
+   $ git checkout v2.5.0
+   $ ./autogen.sh
+   $ ./configure
+   $ make -j4
+   $ make check
+   $ sudo make install
+   $ sudo ldconfig # refresh shared library cache.
```
Comment (ffurrer) [inline README.md]: Do you really need this, can you try with the new instructions on master?

File: rotors_gazebo/resource/fixed_wing/techpod.yaml [hunk 2]
```diff
@@ -12,22 +12,22 @@ throttle_channel: 5
 #   [rad] Maximum deflection angle
 #   Channel number in the array of normalized actuator commands
 aileron_left:
-  channel: 4
+  channel: 0
```
Comment (ffurrer) [inline rotors_gazebo/resource/fixed_wing/techpod.yaml]: should `aileron_left` and `aileron_right` be on the same channel?
probably @hitimo you should take a look at this :)

**Pull request closed without merging**